### DISCUSSION TITLE
Configure alert to send resolved messages

### DISCUSSION THREAD

**Discussion by vikram0602** [grafana/grafana#65564] [Alerting]
Is there a way to configure alert to enable or disable triggering resolved messages.
So for example we want to send resolve messages for some alerts but not for all alerts.

I see you can enable triggering resolved messages on contact points but would be great to set that on alert itself.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82781. Feel free to continue the discussion around this there. Thank you!
